docs/start/install.md

```diff
@@ -52,6 +52,16 @@ cd ..
 pip install -e "python[all_hip]"
 ```
 
+Note: For Intel Gaudi2/3 accelerator, do following instead:
+
+1. Follow the instructions in [Intel Gaudi Installation Guide](https://github.com/HabanaAI/vllm-fork/releases/tag/v0.6.6.post1%2BGaudi-1.20.0) to install the dependencies.
```

Comment:
How about we direct install HabanaAI vllm-fork using pip install git+http://....:commit_id

Reply:
added